src/components/layout/GridDryQueries.jsx

```diff
@@ -0,0 +1,169 @@
+import {device} from '../../data/Devices';
+import styled from 'styled-components'
+const IndexWrapper = styled.div`
+{
+  @media ${device.mobile}
+  {
+      &&&

```

Comment:
What does this do exactly? Furthermore, you duplicated each individual component class for each mediaquery, which isn't necessary, you just need to mediaquery the grid-template-areas?